client/src/Movies/SavedList.js

```diff
@@ -9,10 +10,14 @@ export default class SavedList extends Component {
     return (
       <div className="saved-list">
         <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
-        ))}
-        <div className="home-button">Home</div>
+        {this.props.list.map(movie => {
+          return (
+            <NavLink to={`/movies/${movie.id}`} key={movie.id} activeClassName='saved-active'>
+              <span className='saved-movie'>{movie.title}</span>
+            </NavLink>
```

Comment:
I'm really impressed that you built out the saved-movie list functionality. You even used NavLinks!